pkg/apis/stork/v1alpha1/snapshotrestore.go

```diff
@@ -37,6 +38,8 @@ const (
 	VolumeSnapshotRestoreStatusSuccessful SnapshotRestoreStatusType = "Successful"
 	// VolumeSnapshotRestoreStatusInProgress for when restore is in progress
 	VolumeSnapshotRestoreStatusInProgress SnapshotRestoreStatusType = "InProgress"
+	// VolumeSnapshotRestoreStatusResyncInProgress for when restore is in sync state
+	VolumeSnapshotRestoreStatusResyncInProgress SnapshotRestoreStatusType = "Resync"
```

Comment:
This is PX specific, stork CRDs shouldn't know about it. Store it as an annotation that PX can understand